lib/api/bulkActivityExtract.js

```diff
@@ -97,11 +97,24 @@ BulkActivityExtract.prototype = {
     });
     return this._connection.get(path, { data: options });
   },
-  fileStream: function (exportId, options) {
-    return new MarketoStream(this.file(exportId, options).then(function (data) {
-      return { result: [data] };
-    }));
+  fileStream: function (exportId) {
+    return new Promise(async (resolve) => {
+      var pathToFile = util.createBulkPath('activities', 'export', exportId, 'file.json');

```

Comment:
the two fileStream functions are very similar, they could be generified

Reply:
fixed here 779ae78